napari/components/camera.py

```diff
@@ -27,15 +28,22 @@ class Camera(EventedModel):
     perspective : float
         Perspective (aka "field of view" in vispy) of the camera (if 3D).
     interactive : bool
-        If the camera interactivity is enabled or not.
+        If the camera mouse pan/zoom is enabled or not.
+        Use the mouse_pan and mouse_zoom attributes instead.
+        This attribute is deprecated since 0.5.0 and should not be used.
```

Comment:
```suggestion
        This attribute is deprecated since 0.5.0 and should not be used.
        Use the mouse_pan and mouse_zoom attributes instead.
```